app/public/config.js

```diff
@@ -10,6 +10,19 @@ let configDir = "";
 let isPortable = false;
 let firstRun = false;
 
+// returns additional server args for starting Kopia server
+function additionalServerArgs(repoID) {
+    const argsFile=path.resolve(globalConfigDir(), `server_args-${repoID}.conf`)
+    log.info(`Checking for additional server args in ${argsFile}`)
+    if(fs.existsSync(argsFile)){
```

Comment:
I have couple comments:

1. Can this be a JSON file for future extensibility, also to match all other config files? this will make parsing easier.

```
{
   "extraArgs": [
     "--foo",
     "--bar"
   ]
}
```

2. Later we can set additional things like restart policies, back-off on failure, specify custom-built `kopia` path, pass environment variables and more. 

3. Can make the filename  be `{config-name}.server`?